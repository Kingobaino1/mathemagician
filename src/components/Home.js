import React from 'react';
import Nav from './Nav';

// eslint-disable-next-line arrow-body-style
const Home = () => {
  return (
    <>
      <div className="bg-image">
        <Nav />
      </div>
    </>
  );
};

export default Home;
