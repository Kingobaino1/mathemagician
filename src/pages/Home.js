import React from 'react';
import Nav from '../components/Nav';

// eslint-disable-next-line arrow-body-style
const Home = () => {
  return (
    <>
      <div className="bg-image">
        <Nav />
      </div>
    </>
  );
};

export default Home;
