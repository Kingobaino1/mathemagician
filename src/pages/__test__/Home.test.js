import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from '../Home';

describe('Home Page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
