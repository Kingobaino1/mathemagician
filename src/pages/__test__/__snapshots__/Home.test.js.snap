// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should match the snapshot 1`] = `
<div
  className="bg-image"
>
  <div
    className="d-flex justify-content-between"
  >
    <ul
      className="d-none d-md-block m-4"
    >
      <h1
        className="text-dark font-weight-bold"
      >
        Mathemagician
      </h1>
    </ul>
    <ul
      className="nav m-4"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link active text-dark font-weight-bold"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-dark font-weight-bolder"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-dark font-weight-bold"
          href="/quotes"
          onClick={[Function]}
        >
          Quotes
        </a>
      </li>
    </ul>
  </div>
</div>
`;
